src/lib/themes-test.js

```diff
@@ -33,6 +33,62 @@ describe("lib/themes", () => {
         expect(output).to.deep.equal(theme);
       });
     });
+    it("should migrate themes without popup/popup_text", () => {
+      const theme = {
+        images: {
+          headerURL: ["dummy.png"],
```

Comment:
shouldn't images.headerURL also be in the expected theme along with "images.theme_frame"?

Reply:
Yes. Apparently I need one of the files in `src/images/patterns` for the test to pick it up. I've updated the test.